libs/blocks/src/lib/fast-payment/standard/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { FluidContainer, Heading, Text } from '@deriv/quill-design';
+import {
+  CardsContainer,
+  CardsContainerProps,
+  CustomLink,
+} from '@deriv-com/components';
+import { StandaloneChevronRightRegularIcon } from '@deriv/quill-icons';
+import { LinkProps } from '@deriv-com/utils';
+
+/* eslint-disable-next-line */
+export interface FastPaymentProps {
+  title: string;
+  description?: string;
+  link?: LinkProps;
+  disclaimer?: string;
+  content: Omit<CardsContainerProps<'ContentBottom'>, 'variant'>;
+}
+
+const FastPayment: React.FC<FastPaymentProps> = ({
+  title,
+  description,
+  link,
+  disclaimer,
+  content: { cards = [], cols = 'three', dense },
+}) => {
+  return (
+    <FluidContainer className="bg-background-primary-container py-general-3xl">
+      <div className="flex flex-col items-center">
+        <Heading.H2 className="mb-1200">{title}</Heading.H2>
+        {description && (
+          <Text size="xl" className="mb-gap-md">
```

Comment:
same here I guess can use `gap` instead of `margin-bottom`

Reply:
Same reasoning above.